scrapers/Brazzers.yml

```diff
@@ -205,4 +217,23 @@ xPathScrapers:
 #        fixed: "Female"
       URL: //link[@rel="canonical"]/@href
 
-# Last Updated November 8, 2020
+  galleryScraper:
```

Comment:
do you mind moving the `galleryScraper` section before the performer one and after the scene ? just for cosmetic reasons